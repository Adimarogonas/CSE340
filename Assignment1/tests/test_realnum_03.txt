1.a
2.14
