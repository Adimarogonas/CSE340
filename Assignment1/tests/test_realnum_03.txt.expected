{1 , NUM , 1}
{ , DOT , 1}
{a , ID , 1}
{2.14 , REALNUM , 2}
{ , END_OF_FILE , 4}
